### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by acrobat**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT

I saw your https://github.com/park-manager/hubkit/pull/78#issuecomment-619391060 so I've fixed the test and codestyle issues. This should make most of the test suite pass.

There is one last error in phpstan, but I don't understand why it is an error. Should I ignore it?

### CODE REVIEW

File: tests/Handler/ReleaseHandlerTest.php [hunk 1]
```diff
@@ -155,6 +156,8 @@ public function setUpCommandHandler()
     /** @test */
     public function it_creates_a_new_release_for_current_branch_without_existing_tags()
     {
+        $this->git->getLastTagOnBranch()->willThrow(ProcessFailedException::class);
```
Comment (sstok) [inline tests/Handler/ReleaseHandlerTest.php:159]: Good catch. I actually solved it differently in #81 but this in fact the correct solution 👍 

Can you patch `expectTags` with the following?

```php
    private function expectTags(array $tags = []): void
    {
        $process = $this->prophesize(Process::class);
        $process->getOutput()->willReturn(implode("\n", $tags));

        $this->process->mustRun('git tag --list')->willReturn($process->reveal());

        if ($tags === []) {
            $this->git->getLastTagOnBranch()->willThrow(ProcessFailedException::class);
        } else {
            $this->git->getLastTagOnBranch()->willReturn(end($tags));
        }
    }
```

And revert this change https://github.com/park-manager/hubkit/pull/81/files#diff-307527f4456838fd7b83c398aefbe6aeR186 (as it's actually wrong 🤦 )

Thanks.

**Comment by acrobat** [comment]
@sstok done 👍

**Comment by acrobat** [comment]
@sstok Do you understand the remaining phpstan error? Or should I add an ignore line?

**Comment by sstok** [comment]
Add an ignore 👍 it looks like a false positive

**Comment by acrobat** [comment]
@sstok everything green 🎉

**Comment by sstok** [comment]
Thank you @acrobat

**Comment by acrobat** [comment]
@sstok wdyt, should I try to reach phpstan level 6?
